stoobly_agent/app/cli/scaffold/service_workflow_validate_command.py

```diff
@@ -118,20 +119,22 @@ def validate_internal_hostname(self, url: str) -> None:
     print(f"Validating hostname inside Docker network, url: {url}")
     
     # See WorkflowRunCommand for how 'network' is generated
+    # Debug command=f"curl -k --max-time {timeout_seconds} {url} --verbose",
     network = f"{self.workflow_name}.{self.app.network}" 
     timeout_seconds = 1
+    http_code_format = '"%{http_code}"'
     output = self.docker_client.containers.run(
       image='curlimages/curl:8.11.0',
-      command=f"curl --max-time {timeout_seconds} {url} --verbose",
-      network=APP_EGRESS_NETWORK_TEMPLATE.format(network=network),
+      command=f"curl -k -s -o /dev/null -w {http_code_format} --max-time {timeout_seconds} {url}",
```

Comment:
Do we want `-k`? I think we would want to keep the cert verification

For `-o /dev/null`, would that still give us output to check the response code?

Reply:
If we want to keep -k, then the curl container would need to trust the generated CA certs. Yup, `-o` would

Reply:
@matlegit good question. The detached check was removed from the if condition in this PR. This actually improves the workflow validation